master/pkg/model/task.go

```diff
@@ -20,13 +14,21 @@ func NewAllocationID(name string) AllocationID {
 
 // Allocation is the model for an allocation in the database.
 type Allocation struct {
-	TaskID       TaskID       `db:"task_id"`
 	AllocationID AllocationID `db:"allocation_id"`
+	TaskID       TaskID       `db:"task_id"`
+	Slots        int          `db:"slots"`
+	AgentLabel   string       `db:"agent_label"`
 	ResourcePool string       `db:"resource_pool"`
 	StartTime    time.Time    `db:"start_time"`
 	EndTime      *time.Time   `db:"end_time"`
 }
 
+// TaskID is the unique ID of a task among all tasks.
```

Comment:
Put these above the struct, next to `AllocationID`/`NewAllocationID`?

Reply:
I was just keeping all task stuff together and all allocation stuff together in the file

Reply:
I did ^ this reordering in https://github.com/determined-ai/determined/pull/2938/files already so I'll just leave it as it and then resolve conflicts to keep it that way, if that's ok. It should land shortly too.